README.md

```diff
@@ -59,6 +59,11 @@ image: /path/to/cover/image.jpg # Can be absolute or relative
 ---
 ```
 
+### End
+The end slide is automatically placed at the end by sli.dev. You don't have to add it yourself.
```

Comment:
Since the user has no control over it at all, I would not even name it here. No screenshot, no mention at all

Reply:
Good point!